internal/transport/http/handler.go

```diff
@@ -246,6 +246,7 @@ func groupV1Routes(e *echo.Group, h *Handler) {
 	playlistGroup.POST("/:id", h.AddVodToPlaylist, auth.GuardMiddleware, auth.GetUserMiddleware, auth.UserRoleMiddleware(utils.EditorRole))
 	playlistGroup.DELETE("/:id/vod", h.DeleteVodFromPlaylist, auth.GuardMiddleware, auth.GetUserMiddleware, auth.UserRoleMiddleware(utils.EditorRole))
 	playlistGroup.DELETE("/:id", h.DeletePlaylist, auth.GuardMiddleware, auth.GetUserMiddleware, auth.UserRoleMiddleware(utils.EditorRole))
+	playlistGroup.PUT("/:id/delay", h.SetVodDelayOnPlaylist, auth.GuardMiddleware, auth.GetUserMiddleware, auth.UserRoleMiddleware(utils.EditorRole))
```

Comment:
```suggestion
	playlistGroup.PUT("/:id/multistream/delay", h.SetVodDelayOnPlaylist, auth.GuardMiddleware, auth.GetUserMiddleware, auth.UserRoleMiddleware(utils.EditorRole))
```
Adding `multistream` to the URL makes it move obvious what the delay is for. You will also need to update the frontend API call to this.

Reply:
Updated!